src/System.Private.CoreLib/shared/System/Number.Formatting.cs

```diff
@@ -1205,10 +1205,8 @@ private static unsafe void UInt32ToNumber(uint value, ref NumberBuffer number)
         {
             while (--digits >= 0 || value != 0)
             {
-                // TODO https://github.com/dotnet/coreclr/issues/3439
-                uint newValue = value / 10;
-                *(--bufferEnd) = (byte)(value - (newValue * 10) + '0');
-                value = newValue;
+                value = Math.DivRem(value, 10, out uint remainder);
```

Comment:
The TODO comment here was incorrect, did you verify the generated asm here? Before it was div by const (which is transformed into multiply by const reciprocal), but is it still so?

Reply:
Once DivRem is inlined it becomes the same expression as it was, this change is mostly cosmetic